git/httpconn.go

```diff
@@ -337,6 +339,7 @@ func (s *smartHTTPConn) sendRequest(expectedmime string) error {
 	}
 
 	s.lastresp = resp.Body
+	fmt.Printf("Setting connection")
```

Comment:
Was this meant to be log.Printf()?

Reply:
No, it's meant to be deleted, I just missed it..